hr_skill/views/hr_employee.xml

```diff
@@ -0,0 +1,34 @@
+<openerp>
+  <data>
+
+    <record model="ir.ui.view" id="view_employee_skill_form">
```

Comment:
I think you should inverse model and id
(source : https://www.odoo.com/documentation/8.0/reference/guidelines.html)